internal/models/source_model.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+type Source struct {
+	// Required auto-generated PK.
+	ID uint64 `db:"id" json:"id"`
+
+	// Source ID in sources
+	SourceID string `db:"source_id" json:"source_id"`
```

Comment:
Is this it: https://github.com/RedHatInsights/sources-api-go/blob/main/model/source.go#L24

If so, that is a uint64 why this is a string in our database?